source/_integrations/rehau_nea_smart_manager.markdown

```diff
@@ -0,0 +1,64 @@
+---
+title: "Rehau Nea Smart Manager"
+description: ""
+ha_release: "0.0.1"
+ha_category: Climate
+ha_iot_class: "Local Polling"
+ha_quality_scale: internal
```

Comment:
This is not an internal component of Home Assistant.

Reply:
Fixed.